openshift/cirras-underwriting-ui-deployment.yaml

```diff
@@ -23,6 +23,21 @@ spec:
             - containerPort: #{TARGET_PORT}#
               protocol: TCP
           env:
+            - name: TOMCAT_PORT
+              valueFrom: 
+                configMapKeyRef:
+                  name: cirras-underwriting-api-config-#{ENV}#
```

Comment:
These three settings are referencing the wrong config map: should be UI, not API.

Reply:
Thanks for the catch! Fixed this